src/pages/ReimbursementAccount/ACHContractStep.js

```diff
@@ -13,9 +13,12 @@ const propTypes = {
 
 function ACHContractStep({onBackButtonPress, onCloseButtonPress}) {
     const [isBeneficialOwnerInfoSet, setIsBeneficialOwnerInfoSet] = useState(false);
+    const handleCompleteVerificationBackButtonPress = useCallback(() => {
+        setIsBeneficialOwnerInfoSet(false);
+    }, []);
 
     if (isBeneficialOwnerInfoSet) {
-        return <CompleteVerification setIsBeneficialOwnerInfoSet={setIsBeneficialOwnerInfoSet} />;
+        return <CompleteVerification onBackButtonPress={handleCompleteVerificationBackButtonPress} />;
```

Comment:
onBackButton press should come from above via props. It got updated in all the other steps somewhere along the way. Also you need to pass handler and display `X` button on the header inside CompleteVerification component. It should be similar to BeneficialOwnerInfo